ui/ui-frontend/projects/pastis/src/app/profile/profile-preview/profile-information-tab/profile-information-tab/profile-information-tab.component.html

```diff
@@ -2,9 +2,12 @@
   <div class="row">
     <div class="col-6">
       <vitamui-common-slide-toggle [formControl]="statusProfile" [checked]="form.controls.status.value === 'ACTIVE'">
-        {{ 'PROFILE_PREVIEW.INFORMATION_TAB.PROFILE' | translate }}
-        <span *ngIf="form.controls.status.value === 'INACTIVE'">in</span>
-        {{ 'PROFILE_PREVIEW.INFORMATION_TAB.ACTIF' | translate }}
+        <ng-container *ngIf="form.controls.status.value === 'ACTIVE'; else inactive">
+          {{ 'PROFILE_PREVIEW.INFORMATION_TAB.ACTIVE_PROFILE' | translate }}
+        </ng-container>
+        <ng-template #inactive>
+          {{ 'PROFILE_PREVIEW.INFORMATION_TAB.INACTIVE_PROFILE' | translate }}
+        </ng-template>
```

Comment:
Utiliser la "nouvelle" syntaxe Angular (code à vérifier) :
```suggestion
        @if (form.controls.status.value === 'ACTIVE') {
          {{ 'PROFILE_PREVIEW.INFORMATION_TAB.ACTIVE_PROFILE' | translate }}
        } @else {
          {{ 'PROFILE_PREVIEW.INFORMATION_TAB.INACTIVE_PROFILE' | translate }}
        }
```

Reply:
@ebernard Done